src/settings_translation_file.cpp

```diff
@@ -263,29 +263,6 @@ fake_function() {
 	gettext("Set to true to enable waving leaves.\nRequires shaders to be enabled.");
 	gettext("Waving plants");
 	gettext("Set to true to enable waving plants.\nRequires shaders to be enabled.");
-	gettext("Dynamic shadows");
-	gettext("Dynamic shadows");
-	gettext("Set to true to enable Shadow Mapping.\nRequires shaders to be enabled.");
-	gettext("Shadow strength");
-	gettext("Set the shadow strength.\nLower value means lighter shadows, higher value means darker shadows.");
-	gettext("Shadow map max distance in nodes to render shadows");
-	gettext("Maximum distance to render shadows.");
-	gettext("Shadow map texture size");
-	gettext("Texture size to render the shadow map on.\nThis must be a power of two.\nBigger numbers create better shadows but it is also more expensive.");
-	gettext("Shadow map texture in 32 bits");
-	gettext("Sets shadow texture quality to 32 bits.\nOn false, 16 bits texture will be used.\nThis can cause much more artifacts in the shadow.");
-	gettext("Poisson filtering");
-	gettext("Enable Poisson disk filtering.\nOn true uses Poisson disk to make \"soft shadows\". Otherwise uses PCF filtering.");
-	gettext("Shadow filter quality");
-	gettext("Define shadow filtering quality.\nThis simulates the soft shadows effect by applying a PCF or Poisson disk\nbut also uses more resources.");
-	gettext("Colored shadows");
-	gettext("Enable colored shadows.\nOn true translucent nodes cast colored shadows. This is expensive.");
-	gettext("Map shadows update frames");
-	gettext("Spread a complete update of shadow map over given amount of frames.\nHigher values might make shadows laggy, lower values\nwill consume more resources.\nMinimum value: 1; maximum value: 16");
-	gettext("Soft shadow radius");
-	gettext("Set the soft shadow radius size.\nLower values mean sharper shadows, bigger values mean softer shadows.\nMinimum value: 1.0; maximum value: 10.0");
-	gettext("Sky Body Orbit Tilt");
-	gettext("Set the tilt of Sun/Moon orbit in degrees.\nValue of 0 means no tilt / vertical orbit.\nMinimum value: 0.0; maximum value: 60.0");
```

Comment:
Thse should not be removed

Reply:
This was updated by `generate_from_settingtypes.lua`.  These strings don't seem to actually be used.

Reply:
Nevermind, they're been re-added.  I rebased and re-ran the generator.